ObedZac/iREPORTER
# Ft edit specific redflag 162220527

# 162220527 edit a specific red flag end point

### *What does this PR do*?

This PR creates the end point for the API to edit a specific red flag
the test for editing a red flag should pass

### *Description of Task to be completed*?

The app should edit a specific red flag

### *How should this be manually tested*?

create a virtual environment 
activate the virtual environment
install requirements using pip install -r requirements.txt
run pytest and the test for editing a specific red flag should pass
or run the app via python run.py / flask run and test on postman

### *What are the relevant pivotal tracker stories*?

162220527
